src/nn/models.py

```diff
@@ -361,4 +361,13 @@ def set_covariate_weights(self, weights: numpy.ndarray):
         #('covariate weight shapes are ', weight.shape, snp_count, cov_weights.shape, weight[:, snp_count:].shape)
         weight[:, snp_count:] = cov_weights
         self.layer.weight = torch.nn.Parameter(weight)
-    
\ No newline at end of file
+
+class LassoNetClassifier(LassoNetRegressor):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        
+    def calculate_loss(self, y_hat: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
+        y = y.unsqueeze(1).tile(dims=(1, self.hidden_size))
+        y -= 1 # BCE expects targets in the range [0, 1]
+        y_hat -= 1
```

Comment:
do we need to subtract 1 here as well? I thought it should be two values for probabilities for 2 output neurons, no?